apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/logs/_components/DocumentLogs/DocumentLogInfo/Evaluations.tsx

```diff
@@ -0,0 +1,111 @@
+  useCurrentProject,
+} from '@latitude-data/web-ui'
+import {
+  evaluationMetadataTypes,
+  evaluationResultTypes,
+} from '$/app/(private)/evaluations/_components/ActiveEvaluations/Table'
+import { useCurrentDocument } from '$/app/providers/DocumentProvider'
+import { ROUTES } from '$/services/routes'
+import Link from 'next/link'
+
+import { MetadataItem } from '../../../../_components/MetadataItem'
+import { ResultCellContent } from '../../../../evaluations/[evaluationId]/_components/EvaluationResults/EvaluationResultsTable'
+
+function EvaluationResultItem({ result, evaluation }: ResultWithEvaluation) {
+  if (result.result === undefined) {
+    return <Badge variant='secondary'>Pending</Badge>
+  }
+
+  if (result.resultableType === EvaluationResultableType.Text) {
+    return (
+      <Text.H5 align='left' color='foregroundMuted'>
+        {result.result || '-'}
+      </Text.H5>
+    )
+  }
+
+  return <ResultCellContent evaluation={evaluation} value={result.result} />
+}
+
+export function DocumentLogEvaluations({
+  results,
+}: {
+  results: ResultWithEvaluation[]
+}) {
+  const { project } = useCurrentProject()
+  const { commit } = useCurrentCommit()
+  const document = useCurrentDocument()
+
+  if (!results.length) {
+    return (
+      <Text.H5 color='foregroundMuted' centered>
+        There are no evaluation results for this log
+      </Text.H5>
+    )
+  }
+
+  return (
+    <ul className='flex flex-col gap-4'>
+      {results.map(({ result, evaluation }, index) => (
+        <li key={result.uuid} className='flex flex-col gap-2'>
+          <Text.H4M noWrap ellipsis>
+            {evaluation.name}
+          </Text.H4M>
+          <div className='flex flex-col gap-2.5'>
+            <MetadataItem
+              stacked
+              label='Type'
+              value={`${evaluationResultTypes[evaluation.resultType]} - ${evaluationMetadataTypes[evaluation.metadataType]}`}
+            />
+            <MetadataItem stacked label='Result'>
+              <EvaluationResultItem result={result} evaluation={evaluation} />
+            </MetadataItem>
+            <MetadataItem
+              stacked
+              label='Reasoning'
+              value={result.reason || '-'}
+            />
+          </div>
+          <div
+            className={cn('w-full flex justify-center', {
+              '-mt-2': !result.reason,
+            })}
+          >
+            <Link
+              href={`${
+                ROUTES.projects
+                  .detail({ id: project.id })
+                  .commits.detail({ uuid: commit.uuid })
+                  .documents.detail({ uuid: document.documentUuid })
+                  .evaluations.detail(evaluation.id).root
+              }?resultUuid=${result.uuid}`}
+              target='_blank'
+            >
+              <Button variant='link'>
+                Check evaluation result
+                <Icon
+                  name='externalLink'
+                  className='ml-1'
```

Comment:
This is not the responsability of the children. Put a padding in the parent

Reply:
to give a bit more context: margins usually cause unexpected UI shenanigans so we try to avoid them as much as possible. Usually, you should be able to replicate anything you are trying to do with margins, with  a flexbox container + paddings.